src/BotwinExtensions.cs

```diff
@@ -130,6 +137,10 @@ private static void ApplyGlobalBeforeHook(IApplicationBuilder builder, BotwinOpt
             }
         }
 
+        /// <summary>
+        /// Adds Botwin to the specified Microsoft.AspNetCore.Builder.IApplicationBuilder with the specified Microsoft.AspNetCore.Routing.IRouter .
+        /// </summary>
+        /// <param name="assemblies">Optional assemblies <see cref="Assembly"/> to add to the services collection. If assemblies are not provided, Assembly.GetEntryAssembly is called.</param>
```

Comment:
`Optional params <see cref="Assembly"/> for Botwin to scan. If not provided, Assembly.GetEntryAssembly is called.`